### ISSUE TITLE
rxjs setTimeout error. Causing after successful login

### ISSUE THREAD

**Issue by mitali-gupta** [linkfy/Tools-for-Instagram#27]
After login. When I am performing any task.
It is giving this error and crashing the server.
`
2020-06-14T06:54:42.726346+00:00 app[worker.1]: /app/node_modules/rxjs/internal/
util/hostReportError.js:4
2020-06-14T06:54:42.726362+00:00 app[worker.1]: setTimeout(function () { throw e
rr; }, 0);
2020-06-14T06:54:42.726363+00:00 app[worker.1]: ^
2020-06-14T06:54:42.726364+00:00 app[worker.1]:
2020-06-14T06:54:42.726364+00:00 app[worker.1]: ClientDisconnectedError: MQTToTC
lient got disconnected.
2020-06-14T06:54:42.726365+00:00 app[worker.1]: at SafeSubscriber.client.$discon
nect.subscribe [as _next] (/app/node_modules/instagram_mqtt/dist/realtime/realti
me.client.js:152:30)
2020-06-14T06:54:42.726366+00:00 app[worker.1]: at SafeSubscriber.__tryOrUnsub (
/app/node_modules/rxjs/internal/Subscriber.js:205:16)
2020-06-14T06:54:42.726367+00:00 app[worker.1]: at SafeSubscriber.next (/app/nod
e_modules/rxjs/internal/Subscriber.js:143:22)
2020-06-14T06:54:42.726367+00:00 app[worker.1]: at Subscriber._next (/app/node_m
odules/rxjs/internal/Subscriber.js:89:26)
2020-06-14T06:54:42.726368+00:00 app[worker.1]: at Subscriber.next (/app/node_mo
dules/rxjs/internal/Subscriber.js:66:18)
2020-06-14T06:54:42.726368+00:00 app[worker.1]: at Subject.next (/app/node_modul
es/rxjs/internal/Subject.js:60:25)
2020-06-14T06:54:42.726369+00:00 app[worker.1]: at MQTToTClient.setDisconnected
(/app/node_modules/mqtts/dist/mqtt.client.js:329:30)
2020-06-14T06:54:42.726369+00:00 app[worker.1]: at Object.disconnect (/app/node_
modules/mqtts/dist/mqtt.client.js:97:26)
2020-06-14T06:54:42.726370+00:00 app[worker.1]: at TLSSocket.socket.on (/app/nod
e_modules/mqtts/dist/transport/tls.transport.js:20:52)
2020-06-14T06:54:42.726370+00:00 app[worker.1]: at TLSSocket.emit (events.js:194
:15)
2020-06-14T06:54:42.726371+00:00 app[worker.1]: at endReadableNT (_stream_readab
le.js:1103:12)
2020-06-14T06:54:42.726371+00:00 app[worker.1]: at process._tickCallback (intern
al/process/next_tick.js:63:19)
2020-06-14T06:54:42.726372+00:00 app[worker.1]: Emitted 'error' event at:
2020-06-14T06:54:42.726372+00:00 app[worker.1]: at RealtimeClient.emitError (/ap
p/node_modules/instagram_mqtt/dist/realtime/realtime.client.js:23:38)
2020-06-14T06:54:42.726373+00:00 app[worker.1]: at SafeSubscriber.client.$discon
nect.subscribe [as _next] (/app/node_modules/instagram_mqtt/dist/realtime/realti
me.client.js:152:20)
2020-06-14T06:54:42.726373+00:00 app[worker.1]: at SafeSubscriber.__tryOrUnsub (
/app/node_modules/rxjs/internal/Subscriber.js:205:16)
2020-06-14T06:54:42.726374+00:00 app[worker.1]: [... lines matching original sta
ck trace ...]
2020-06-14T06:54:42.726374+00:00 app[worker.1]: at process._tickCallback (intern
al/process/next_tick.js:63:19)
`

I think problem with request module.

**Comment by Nerixyz** [comment]
Please add `DEBUG=mqtt:*,ig:*` as an environment variable.

**Comment by virginsince1997** [comment]
@Nerixyz it's the same issue with regeneration sessions, I assume they are using the anitban module, it also regenerates sessions, @linkfy is working on it. @mitali-gupta remove setAntiBan if it's in your script

**Comment by mitali-gupta** [comment]
> Please add `DEBUG=mqtt:*,ig:*` as an environment variable.

Not working.

**Comment by mitali-gupta** [comment]
> @Nerixyz it's the same issue with regenerating sessions, I assume they are using the anitban module, it also regenerates sessions, @linkfy is working on it. @mitali-gupta remove setAntiBan if it's in your script

It worked.

**Comment by linkfy** [comment]
Yes, another way is to set online mode to false on the env file, in this way you can use antiban at the moment.

**Comment by Ferdian9991** [comment]
I don't use antiBanMode, but I still get the same error when ONLINE_MODE is set to true
